cli/flags/common.go

```diff
@@ -27,6 +27,7 @@ var (
 	dockerCertPath  = os.Getenv("DOCKER_CERT_PATH")
 	dockerTLSVerify = os.Getenv("DOCKER_TLS_VERIFY") != ""
 	dockerTLS       = os.Getenv("DOCKER_TLS") != ""
+	dockerContext   = os.Getenv("DOCKER_CONTEXT")
```

Comment:
There's a definition for `"DOCKER_CONTEXT"` this in moby/moby/client/client.go, use it?

Reply:
Not anymore